### DISCUSSION TITLE
ejabberd 24.12 is here !

### DISCUSSION THREAD

**Discussion by mremond** [processone/ejabberd#4329] [Announcements]
ejabberd 24.12 is here and we've nicknamed it the "evacuate_kindly" release, inspired by one of its new commands, with a playful twist. As the year winds down, we hope this update helps you smoothly manage your messaging infrastructure—and also reminds you to kindly "evacuate" the stress of 2024.

Wishing you a wonderful holiday season!

More details in our blog post:
https://www.process-one.net/blog/ejabberd-24-12/

**Comment by licaon-kter** [comment]
no s2s bidi? Pffft

**Reply by mremond** [reply]
It was already in ejabberd 24.10: https://www.process-one.net/blog/ejabberd-24-10/#bidi

**Reply by licaon-kter** [reply]
@mremond no, not really: https://github.com/processone/ejabberd/issues/4300

**Reply by mremond** [reply]
Ah right. Thanks for the reminder then :)
